README.md

```diff
@@ -128,14 +128,14 @@ In addition to allowing the storage of strings, Valet allows the storage of `Dat
 ### Sharing Secrets Among Multiple Applications
 
 ```swift
-let mySharedValet = Valet.sharedAccessGroupValet(with: Identifier(nonEmpty: "Druidia")!, accessibility: .whenUnlocked)
+let mySharedValet = Valet.sharedAccessGroupValet(with: Identifier(nonEmpty: "AppID12345.Druidia")!, accessibility: .whenUnlocked)
```

Comment:
I think it might be cleaner to create a distinct `SharedAccessGroupIdentifier` that takes two separate strings (the shared access group id and the Valet id), rather than using `Identifier` here. That way it is more obvious what this identifier is, and consumers don't have to know that it needs to be formatted as two strings concatenated with a period.

Reply:
I like this a lot. Made code changes in 5078ffc9fdf3847757a0003132153d3f302a3dda. Changes to documentation incoming.